src/python/pants/backend/docker/goals/package_image.py

```diff
@@ -225,6 +234,12 @@ async def build_docker_image(
     docker: DockerBinary,
     process_cleanup: ProcessCleanupOption,
 ) -> BuiltPackage:
+    """Build a Docker image.
```

Comment:
Ditto on the value of this rule. The first sentence and last sentence are already expressed by the rule's name and its return type. The middle sentence seems clear from the code itself.

Reply:
The idea is to describe briefly the purpose of the thing, without having to read the code, as you may not have the code in reach when doing `./pants help Thing`

Reply:
Okay. Still, `help` will include the rule name & return type, right? So that means only the middle sentence has some value

You could perhaps shorten to one sentence:

> Build a docker image by using `docker build` ...